src/components/Sidebar/ChoiceNode.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, ChangeEvent } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronRight, faEdit } from '@fortawesome/free-solid-svg-icons';
+import { usePathwayContext } from 'components/PathwayProvider';
+import styles from './Sidebar.module.scss';
+import { FormControl, makeStyles, TextField, InputAdornment, Input } from '@material-ui/core';
+
+interface ChoiceNodeProps {
+  transition: string;
+}
+
+const ChoiceNode: FC<ChoiceNodeProps> = ({ transition }) => {
+  const { pathway } = usePathwayContext();
+
+  const useStyles = makeStyles(theme => ({
+    formControl: {
+      margin: theme.spacing(1, 0),
+      minWidth: 120,
+      display: 'flex',
+      flexWrap: 'wrap'
+    },
+    rename: {
+      fontSize: '2rem',
+      display: 'flex',
+      flexWrap: 'wrap'
+    }
+  }));
+
+  const classes = useStyles();
+  const initial = pathway !== null ? pathway.states[transition].label : 'Choice Node';
+
+  // TODO: in PATHWAYS-256 use the data from the pathway
+  const [choice, setChoice] = React.useState('');
+  const [label, setLabel] = React.useState<string>(initial);
```

Comment:
Remove React here and add type to choice
```suggestion
  const [choice, setChoice] = useState<string>('');
  const [label, setLabel] = useState<string>(initial);
```